tests/e2e/pageobjects/dashboard/CreateWorkspace.ts

```diff
@@ -49,6 +51,12 @@ export class CreateWorkspace {
         await this.driverHelper.waitAndClick(sampleLocator, timeout);
     }
 
+    async startWorkspaceUsingFactory(factoryUrl: string, timeout: number = TimeoutConstants.TS_CLICK_DASHBOARD_ITEM_TIMEOUT) {
+        Logger.debug(`CreateWorkspace.startWorkspaceUsingFactory factoryUrl: "${factoryUrl}"`);
```

Comment:
There is another way to create workspace using factory url, which is already implemented: 
```
await browserTabsUtil.navigateTo(factoryUrl);
```

Does it make sense to support one more way to create workspace using factory URL - from "Create workspace" page, taking into account that it's less reliable because depends on Dashboard GUI?